### ISSUE TITLE
RAGE 2 (548570)

### ISSUE THREAD

**Issue by ghost** [ValveSoftware/Proton#2686]
# Compatibility Report
- Name of the game with compatibility issues: RAGE 2
- Steam AppID of the game: 548570

## System Information
- GPU: GTX 1080 Ti 
- Driver/LLVM version: Nvidia 430.09-41 BETA
- Kernel version: 5.0.12-115
- Link to full system information report as Gist: https://gist.github.com/SquidTheSid/b4e96641391ecc8497344fd7f6dd9a69
- Proton version: 4.2-3

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.

- [X] that I have checked whether there are updates for my system available.
[steam-548570.log](https://github.com/ValveSoftware/Proton/files/3176300/steam-548570.log)

## Symptoms 
Game crashes to desktop after launch, on the photosensitivity warning screen. 

## Reproduction

Select Proton 4.23 in options, launch game, wait for setup to complete, game crashes on photosensitivity warning. If not using Beta drivers, then click "yes" on drivers out of date, proceed anyway message when it pops up.

**Comment by kisak-valve** [comment]
This looks like the line of interest in the log: `wine: Call from 0x7b44c03c to unimplemented function api-ms-win-crt-math-l1-1-0.dll.fmaf, aborting`

**Comment by ntropy83** [comment]
Can confirm the same wine error:

    GPU: Vega 64
    Driver/LLVM version: RADV 19.1, 9.0.0 LLVM and tried too: AMDVLK Q2019.Q2.2 
    Kernel version: 5.0.15-liquorix
    Proton version: 4.2-3

I sometimes get an error dialog about unknown graphics drivers, yet it appears randomly. Tried installing vcrun2015 via winetricks and did it manually, but didnt change the wine error.

**Comment by ghost** [comment]
> but didnt change the wine error.

Nothing you do should change that error. The flaw with Wine is that if a game comes out that uses something Wine doesn't already implement to some degree, then it crashes and can't be used until its implemented to the needed degree.

On the other side of the coin is that there are errors that do not keep games from working to the game's end but could still introduce various problems.

So if this error is what's keeping the game from working and it very much appears to be then you can't do squat to fix it unless you want to do the programming yourself. So you can wait until someone does instead.

Maybe though if you can get it the .dll and it accepts it as a workaround. You have to make sure its not taking in the Wine dll version and if still not working then its a most likely a done deal.

In this case it looks like its only finding "C:\\windows\\system32\\api-ms-win-crt-math-l1-1-0.dll" is a fake Wine dll https://github.com/wine-mirror/wine/tree/master/dlls/api-ms-win-crt-math-l1-1-0 > stub fmaf

#### And Valve decided to add in a fix https://github.com/ValveSoftware/Proton/releases/tag/proton-4.2-4 though possibly not totally fixed. It looks like there may be other problems too.

**Comment by trenton4k** [comment]
random crashes going to map

**Comment by ntropy83** [comment]
Can get in game now but sometimes it still hangs on the loading screen.
Overall 25 fps more on AMDVLK than on RADV.
Thx for the fix so far :)

**Comment by kisak-valve** [comment]
Hello @trenton4k, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the crash, and drag and drop the generated $HOME/steam-$APPID.log into the comment box. Also, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by BoldyPlays** [comment]
Hey Kisak, in my case its not launching, despite the update to 4.2-4. Captured the information as you requested in your response to trenton.

[steam-548570.log](https://github.com/ValveSoftware/Proton/files/3182627/steam-548570.log)

Steam system information : https://gist.github.com/BoldyPlays/d1139dcfdb9c3f7fe9f557d2e5b472fa

**Comment by kisak-valve** [comment]
Hello @BoldyPlays, the [Proton 4.2-4](https://github.com/ValveSoftware/Proton/releases/tag/proton-4.2-4) release notes does mention `Fix crash on launch for RAGE 2. (requires Mesa development builds on AMD)` for this game, so mesa 19.0.3 / llvm 7 is most likely not new enough for initial support.

**Comment by BoldyPlays** [comment]
Hey @kisak-valve good shout, I'll take care of that immediately - Appreciated!

**Comment by BoldyPlays** [comment]
Tried to use something a little more recent @kisak-valve to no avail, same result. I've refunded the game for now, maybe revisit it when AMD drivers are not a total cluster**** - cheers for getting back to me man!

https://gist.github.com/BoldyPlays/d1139dcfdb9c3f7fe9f557d2e5b472fa

**Comment by trenton4k** [comment]
https://gist.github.com/trenton4k/300c294e4d8b13988b81c5e45f73fe2a

> Hello @trenton4k, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options, reproduce the crash, and drag and drop the generated $HOME/steam-$APPID.log into the comment box. Also, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

https://gist.github.com/trenton4k/300c294e4d8b13988b81c5e45f73fe2a

[steam-548570.log](https://github.com/ValveSoftware/Proton/files/3186702/steam-548570.log)

**Comment by ntropy83** [comment]
I have the same issues on AMD, on RADV the game crashes on the epilepsy warning screen, on AMDVLK it does run every other try or crashes when the Bethesda Logo starts to show. After 3.5 hours of play I got a random crash to desktop midgame. 
I will post my sys infos later, am at work right no and log the game. If I can catch a midgame exit, I will post the proton log.
https://gist.github.com/enthalpie/a336f437376b7912305fa381d91d3b7e
P.S.: its working now pretty good on AMDVLK

**Comment by NTMan** [comment]
The game works with RADV too, but crashes every time when I change resolution from FHD to UHD.

![Screenshot from 2019-05-19 00-34-58](https://user-images.githubusercontent.com/200750/57974242-679bd300-79ce-11e9-85e3-a162aaf44947.png)

[steam-548570.log](https://github.com/ValveSoftware/Proton/files/3194424/steam-548570.log)
[steam-548570.log](https://github.com/ValveSoftware/Proton/files/3194429/steam-548570.log)


```
$ inxi -bM
System:    Host: localhost.localdomain Kernel: 5.1.0-1.fc31.x86_64 x86_64 bits: 64 Desktop: Gnome 3.33.1 
           Distro: Fedora release 31 (Rawhide) 
Machine:   Type: Desktop Mobo: ASUSTeK model: ROG STRIX X470-I GAMING v: Rev 1.xx serial: <root required> 
           UEFI: American Megatrends v: 2202 date: 04/11/2019 
CPU:       8-Core: AMD Ryzen 7 2700X type: MT MCP speed: 2195 MHz min/max: 2200/3700 MHz 
Graphics:  Device-1: Advanced Micro Devices [AMD/ATI] Vega 20 [Radeon VII] driver: amdgpu v: kernel 
           Display: wayland server: Fedora Project X.org 1.20.4 driver: amdgpu resolution: 3840x2160~60Hz 
           OpenGL: renderer: AMD Radeon VII (VEGA20 DRM 3.30.0 5.1.0-1.fc31.x86_64 LLVM 8.0.0) v: 4.5 Mesa 19.1.0-rc2 
Network:   Device-1: Intel I211 Gigabit Network driver: igb 
           Device-2: Realtek RTL8822BE 802.11a/b/g/n/ac WiFi adapter driver: r8822be 
Drives:    Local Storage: total: 11.57 TiB used: 8.02 TiB (69.3%) 
Info:      Processes: 456 Uptime: 21m Memory: 31.40 GiB used: 14.75 GiB (47.0%) Shell: bash inxi: 3.0.34 
```

**Comment by xpander69** [comment]
Game will Freeze with nvidia from time to time, when opening tab menu or esc menu. Having vsync enabled makes it happening less, but still happens from time to time.

- **GPU:** GTX 1080 Ti
- **Driver/LLVM version:** Nvidia 430.14
- **Kernel version:** 5.1.4-zen
- **Link to full system information report as Gist:** https://gist.github.com/xpander69/8c756b27f6f28037072c2e8adaf2216d
- **Proton version:** 4.2-5
- **Proton log:** [steam-548570.log](https://github.com/ValveSoftware/Proton/files/3219461/steam-548570.log)

edit: Turned off Async Compute from `~/SteamLibrary/steamapps/compatdata/548570/pfx/drive_c/users/steamuser/Saved Games/id Software/Rage 2/settings.ini `

2+ hours without a freeze now. If anyone else can also confirm it? might be just luck though :)

edit2: 8+ hours in a row without issues, when Async Compute is disabled.

**Comment by Lamieur** [comment]
Confirmed on another 1080 Ti - AsyncCompute=0 fixes triple-tab crash. Thank you!

**Comment by supertriodo** [comment]
Confirmed on a GTX 1060 AsyncCompute=0 fixes the crash related with the tab menu.

**Comment by alutarius** [comment]
Still randomly crashes for me when opening the Tab menu with AsyncCompute=0 in settings.ini on RTX 2080 with physical copy activated on bethesda.net with Lutris and Proton 4.2-7.

**Comment by ghost** [comment]
The new patch seems to break the game. It fails to launch and a "RAGE2 has stopped working" window appears every time I launch the game. 

EDIT: Reinstalling the game after the patch seemed to fix the issue. I think it was an issue with an environment variable.

**Comment by xpander69** [comment]
Works fine for me, without reinstall. Only thing was that the resolution and graphical settings were all reset. So i had to configure them again to disable the stupid blur, chromatic aberration and the washed out AA this game has.

**Comment by Vash63** [comment]
Is anyone else getting microstuttering on mouse movement? When I move _only_ with the keyboard it is silky smooth. Same test area moving only the mouse slowly exhibits a consistent stutter like twice a second. I may try to get a video to show it but I'm not sure if my camera is fast enough that it will be visible.

I'm getting 70+ FPS constantly and I'm on a gsync display so I'm fairly sure it's not an actual framerate drop, just feels like something is being presented out of order or twice or something.

**Comment by Vash63** [comment]
![20190922230405_1](https://user-images.githubusercontent.com/3086017/65394418-28233000-dd8e-11e9-88c5-2b890d2a19d1.jpg)

Stuttering solved - disabling the chroma effects option above resolved the issue completely (it's on by default). It's smooth as butter now. Not sure if that would be enough for whitelisting as you have to change a setting but I doubt Chroma is a dealbreaker for many people.

**Comment by kisak-valve** [comment]
RAGE 2 - machine lockup / hangs while loading checkpoints.

Issue transferred from https://github.com/ValveSoftware/Proton/issues/3113.
@rcbrufatto posted on 2019-10-01T19:19:17:

Currently facing an odd behavior within RAGE 2 with Proton 4.11-6.
Game starts and works flawlessly, but usually freezes while loading from last save or checkpoint. The only thing that becomes available is the infamous Alt+SysRq REISUB. 
I have tried a bunch of tweaks as proposed on multiple places also driver updates without luck. Also, I haven't found much relevant information in the logs.

HW info:

```
System:
  Host: xxxxxx Kernel: 5.3.0-13-generic x86_64 bits: 64 
  Desktop: Xfce 4.14.1 Distro: Ubuntu 19.10 (Eoan Ermine) 
Machine:
  Type: Laptop System: Micro-Star product: GS65 Stealth Thin 8RF v: REV:1.0 
  serial: <root required> 
  Mobo: Micro-Star model: MS-16Q2 v: REV:1.0 serial: <root required> 
  UEFI: American Megatrends v: E16Q2IMS.112 date: 05/21/2019 
Battery:
  ID-1: BAT1 charge: 79.8 Wh condition: 83.0/80.3 Wh (103%) 
CPU:
  6-Core: Intel Core i7-8750H type: MT MCP speed: 899 MHz 
  min/max: 800/4100 MHz 
Graphics:
  Device-1: Intel UHD Graphics 630 driver: i915 v: kernel 
  Device-2: NVIDIA GP104M [GeForce GTX 1070 Mobile] driver: nvidia v: 435.21 
  Display: x11 server: X.Org 1.20.5 driver: modesetting,nvidia 
  unloaded: fbdev,nouveau,vesa resolution: 1920x1080~144Hz 
  OpenGL: renderer: GeForce GTX 1070 with Max-Q Design/PCIe/SSE2 
  v: 4.6.0 NVIDIA 435.21 
Network:
  Device-1: Intel Wireless-AC 9560 [Jefferson Peak] driver: iwlwifi 
  Device-2: Qualcomm Atheros Killer E2500 Gigabit Ethernet driver: alx 
Drives:
  Local Storage: total: 931.51 GiB used: 488.34 GiB (52.4%) 
Info:
  Processes: 317 Uptime: 1h 23m Memory: 31.28 GiB used: 2.00 GiB (6.4%) 
  Shell: bash inxi: 3.0.36
```
I am also attaching steamapp logs.

Let me know what other info I can provide. In the meantime, I will keep looking if I can find something more relevant in the logs.

Thanks in advance!

[steam-548570.log](https://github.com/ValveSoftware/Proton/files/3678122/steam-548570.log)

**Comment by rcbrufatto** [comment]
A quick update: 

Setting the key 'AsyncComputeDisable' to 1 on settings.ini seemed to have fixed it!
Will run some more tests late today to make sure.

**Comment by rcbrufatto** [comment]
Setting the key 'AsyncComputeDisable' to 1 on settings.ini indeed solved the issue for me.
Played for a long time (died many times) and it did not crash anymore. Could not reproduce the error.
Hope that helps someone else with the same issue I had.

**Comment by hakzsam** [comment]
@NTMan Are you still able to reproduce the crash when you change the resolution with Mesa 20.1?

**Comment by hakzsam** [comment]
I have just pushed a workaround for a game bug (the white circles issue). Please give it a new try. Thanks!

**Comment by NTMan** [comment]
> @NTMan Are you still able to reproduce the crash when you change the resolution with Mesa 20.1?

I am was able to reproduce the game crash when switch from fullscreen mode to borderless with mesa 20.1.99 (900bf50).
![Screenshot from 2020-06-12 18-29-13](https://user-images.githubusercontent.com/200750/84507748-bf446500-acda-11ea-9a4a-dba7f490606a.png)

**Comment by hakzsam** [comment]
@NTMan Are you also able to reproduce with AMDLVK or AMDGPU-PRO? I tried again and it didn't happen.

**Comment by RomanHargrave** [comment]
Some observations 

Relevant specs:
* i7-4790k
* Radeon VII
* 32GB RAM
* Disp 1 - 2560x1440@144hz+0x0, Disp 2 - 2560x1440@60hz+2560x0
* Kernel `5.6.0-pf4-torvalds-git` (pf-kernel)
* Proton: `5.11-GE-1`

Game issues/quirks:
* RADV performance is not idea (40fps at 1440p compared to 75+fps with AMDVLK), AMDVLK/xgl performance is excellent. 
* The game somewhat predictably crashes after an hour to several hours of gameplay, and I cannot find any specific trigger. 
* AsyncComputeDisable does not appear to mitigate the instability I am seeing, rather it significantly impacts performance and causes extreme stuttering.
* The overlay can permanently steal mouse input some small percentage of the time.

**Comment by Mohandevir** [comment]
Same bug as NTMan reported (Rage 2 stopped, DX Diag...) with AsyncComputeDisable or not, Fullscreen or borderless.
It crashes every time whithin an hour of play (often under 15 mins). Game is unplayable, atm.

Ryzen 5 3600 with GTX 1660 Super on 440.100 driver.
Linux Mint 20

Thanks for your support!

**Comment by kisak-valve** [comment]
Tracking note:

Dropping the AMD RADV / Mesa drivers labels due to outdated feedback from @NTMan. It looks like @hakzsam  was looking for a retest with mesa newer than https://cgit.freedesktop.org/mesa/mesa/commit/?id=e7e9969efec777443a022056939049dfa82c5cf4 (at this point that would be mesa 20.2, git, or [kisak-mesa 20.1.1~kisak2](https://launchpad.net/~kisak/+archive/ubuntu/kisak-mesa) and newer) and https://cgit.freedesktop.org/mesa/mesa/commit/?id=900bf50c39089ef9841eccd729527319eaf49888 is older than that.

@NTMan, if you can reproduce your resolution change issue with an updated system, let me know and I can re-add those labels.

**Comment by jarrard** [comment]
Been trying this lately, things I observed with MESA20 and 21 on 6800XT

1. Stuttering whenever vsync was enabled.
2. ACO compiler corrupts some graphics (menus and maybe some shaders in game).

As for stability, still testing.

**Comment by kisak-valve** [comment]
Hello @jarrard, please copy your system information from Steam (`Steam` -> `Help` -> `System Information`) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report. Also, please include a screen shot of the visual corruption you're seeing.

**Comment by jarrard** [comment]
https://gist.github.com/jarrard/09d53558bb135408994bf85250d09415

In-game shadows and transparencies seem off (not blending right). Main menus have black blocks all over the place and some animated icons and videos.

![rage2 ingame aco](https://user-images.githubusercontent.com/17428910/111687244-fbb07d00-8879-11eb-87ff-55ac2fced771.jpg)

![rage2 menu aco](https://user-images.githubusercontent.com/17428910/111687274-01a65e00-887a-11eb-8a16-f486f57c559c.jpg)

This issue is duplicate of link below, and is apparently fixed in future MESA versions, which will one day appear in mainline mesa _(I just use stable arch/manjaro atm, tried the mesa-git and it didn't resolve the issue so must be older git)._

https://gitlab.freedesktop.org/mesa/mesa/-/issues/4163

**Comment by kisak-valve** [comment]
Thanks, it's not ideal to be reporting graphical issues with vkBasalt active and potentially interfering with the results, but setting that aside ...

The menu rendering issue is https://gitlab.freedesktop.org/mesa/mesa/-/issues/4163, fixed in mesa 21.0.0 and queued for the not-yet-released mesa 20.3.5.

The lighting issue reminds me of https://gitlab.freedesktop.org/mesa/mesa/-/issues/4367 / https://gitlab.freedesktop.org/mesa/mesa/-/issues/4394 which is fixed in mesa 21.0.0, queued for mesa 20.3.5.

I'm going to skip adding the mesa / radv labels on your feedback this time, but let me know if you can reproduce the issue on mesa 20.3.5+ and we can re-evaluate.

**Comment by jarrard** [comment]
I normally turn off vkbasalt etc but forgot this time around. I would test again if there wasn't already a bug report about this issue so will just wait for the next mesa release.

**Comment by zzhiyi** [comment]
For AMD users, you can set Fullscreen=1 in steamapps/compatdata/548570/pfx/drive_c/users/steamuser/Saved Games/id Software/Rage 2/settings.ini to use the borderless window mode to avoid crashes on focus loss.

**Comment by ghost** [comment]
> For AMD users, you can set Fullscreen=1 in steamapps/compatdata/548570/pfx/drive_c/users/steamuser/Saved Games/id Software/Rage 2/settings.ini to use the borderless window mode to avoid crashes on focus loss.

tried this on proton 8 and proton experimental with no luck, still crashes at start

**Comment by kisak-valve** [comment]
RAGE 2 Crashing on TAB menu

Issue transferred from https://github.com/ValveSoftware/Proton/issues/7692.
@Shadow-Hunt3r posted on 2024-05-03T19:00:26:

# Compatibility Report
- Name of the game with compatibility issues: RAGE 2
- Steam AppID of the game: 548570

## System Information
- GPU: NVIDIA GeForce MX450
- Video driver version: NVIDIA 535.171.04
- Kernel version: 5.15.0-105-generic
- Link to full system information report as [Gist](https://gist.github.com/): [Full System Info](https://gist.github.com/Shadow-Hunt3r/0229f458a8ba4c18d416b38657696e53.js)
- Proton version:(Tested the bug on) Experimental, 9.0-1, 8.0-5

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

Experimental - [steam-548570.log](https://github.com/ValveSoftware/Proton/files/15203907/steam-548570.log)
9.0-1 - [steam-548570.log](https://github.com/ValveSoftware/Proton/files/15203910/steam-548570.log)
8.0-5 - [steam-548570.log](https://github.com/ValveSoftware/Proton/files/15203917/steam-548570.log)



## Symptoms 
Game hard crashes to a point where it shows last frame(of whatever you had loaded in the TAB menu) and to close it regardless of windowed, fullscreen or borderless you have to switch to the steam launcher and force stop the game


## Reproduction

1. Install RAGE 2 with any of the 3 proton versions I mentioned above
2. Run the game
3. Open TAB menu, this crash seems to occur a lot when in the wilderness, or mission zones


## Additional Notes
I have also attempted using protontricks, but to no change, then I tried using Virtual Desktop with the aid of WineCFG* to enable it, the frequency of the crashes dropped marginally however

*Along with the marginal change it gave it also worsened the preformance and added audio tearing in fullscreen mode, with windowed and borderless showed better results, those being much less audio tearing and better preformance along with decreasing the frequency of the crashes

**Comment by LethalManBoob** [comment]
Is anyone else having severe performance issues in general? Specifically on nvidia?

**Comment by simifor** [comment]
You'll need to provide more information. When do the performance issues start? main menu, when you go ingame, or after playing for a while? system information? game settings?

I don't have a nvidia card, but after playing for a few minutes with both low and ultra graphics presets with proton 9.0-4 and comparing against windows I got comparable performance. Also couldn't get the game to crash on my side with the tab menu.

**Comment by MagicManUSA** [comment]
It's Ingame. Works fine on Windows. I have an Nvidia RTX 4060. Tried Arch, Fedora, other Linux Distros all the same. Does it on any setting Fullscreen, Borderless, Vsync on and off.

Also to quote this guy from proton db (Copy and Pasted section)

Begin-

Instability:Frequently

Unless I disabled Show Hud, frequent crash every time a Hud element was displayed, especially early game with all the tutorial prompts (each would cause a crash when Hud was enabled).

TLDR: -Something in this games HUD is broken when trying to play on LINUX for me (constant crash) -Show HUD = Off (ran fine and perfectly smooth without crashes)

End-

(Possibly the culprit and apparently on AMD Cards there are no problems on any Linux Distro)

Hope this helps!

**Comment by xpander69** [comment]
So nvidia users if you are eager to play this. There's a way:

Latest patch from 17th sept 2024 made the game unplayable on nvidia. Hangs when opening tab/map menu. AsyncComputeDisable=1 doesnt work anymore. to work around it we need to download the patch before that.

open steam with: 
`steam steam://open/console` 

paste it to the steam console tab: 
`download_depot 548570 548572 1826443442961353435`

then copy and replace the depot downloaded files into the actual game directory

Create a **steam_appid.txt** file with **548570** into the game folder in order to not let the game update.

Change **AsyncComputeDisable** to **1** in
 `~/SteamLibrary/steamapps/compatdata/548570/pfx/drive_c/users/steamuser/Saved Games/id Software/Rage 2/settings.ini`

edit: you also have to cap the fps to 120 either via mangohud or libstrangle. otherwise the map screen loads and hangs if FPS is too high.

Nvidia issue report here:
https://forums.developer.nvidia.com/t/rage-2-native-vulkan-windows-game-hangs-on-nv-linux-drivers-works-on-amd-radv-and-nv-windows-drivers/169083/10

**Comment by xpander69** [comment]
Ok RAGE 2 is finally fixed on nvidia 580.105.08
no more hangs on tab menu

**Comment by LethalManBoob** [comment]
Hello. Rage 2 for me has stuttering if using mangohud or libstrangle to cap the frame rate.
It stops being a problem around 120 fps but 90fps shows issues.
Enabling vsync seems to smooth it out a little. Still suboptimal. Seems to elude mangohud why the stuttering is occuring. Seems to be from the input side. 

Latest nvidia driver 580.105.08
de: kde plasma - wayland
proton versions seem to not change this.

**Comment by xpander69** [comment]
> Hello. Rage 2 for me has stuttering if using mangohud or libstrangle to cap the frame rate. It stops being a problem around 120 fps but 90fps shows issues. Enabling vsync seems to smooth it out a little. Still suboptimal. Seems to elude mangohud why the stuttering is occuring. Seems to be from the input side.


Does this happen with 
**fps_limit_method** **late** or **early**?
in mangohud? maybe difference there. I just recently uninstalled the game to make some free space. no idea what method libstrangle uses though.

**Comment by LethalManBoob** [comment]
> > Hello. Rage 2 for me has stuttering if using mangohud or libstrangle to cap the frame rate. It stops being a problem around 120 fps but 90fps shows issues. Enabling vsync seems to smooth it out a little. Still suboptimal. Seems to elude mangohud why the stuttering is occuring. Seems to be from the input side.
> 
> Does this happen with **fps_limit_method** **late** or **early**? in mangohud? maybe difference there. I just recently uninstalled the game to make some free space. no idea what method libstrangle uses though.

I tried both. Same result.

**Comment by YMan84** [comment]
> So nvidia users if you are eager to play this. There's a way:
> 
Thank you! The bug is still present and this worked for me in 2026.
